net-libs/webkit-gtk/webkit-gtk-2.38.0-r410.ebuild

```diff
@@ -0,0 +1,252 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_REQ_USE="xml(+)"
+PYTHON_COMPAT=( python3_{8..11} )
+USE_RUBY="ruby27 ruby30 ruby31"
+
+inherit check-reqs flag-o-matic gnome2 optfeature python-any-r1 ruby-single toolchain-funcs cmake
+
+MY_P="webkitgtk-${PV}"
+DESCRIPTION="Open source web browser engine"
+HOMEPAGE="https://www.webkitgtk.org"
+SRC_URI="https://www.webkitgtk.org/releases/${MY_P}.tar.xz"
+
+LICENSE="LGPL-2+ BSD"
+SLOT="4.1" # soname version of libwebkit2gtk-4.0
```

Comment:
I suspect we'll want a subslot here to be safe. (Also, fix the mention of `libwebkit2gtk-4.0` to `libwebkit2gtk-4.1`).

I see that the shared objects SONAMEs changed like this:

````diff
-./usr/lib64/libjavascriptcoregtk-4.0.so.18.21.3
-./usr/lib64/libwebkit2gtk-4.0.so.37.57.3
+./usr/lib64/libjavascriptcoregtk-4.1.so.0.2.3
+./usr/lib64/libwebkit2gtk-4.1.so.0.2.3
````

So I think it should be `SLOT="4.1/0".

Reply:
I don't expect any ABI breaks here anymore, it was a one-time exceptional thing back in CVS times. It remained 4/37 for the whole time in git.
I don't mind either way for 4.1, but I am 99,99% sure it won't change ABI for the gtk4 version that's planned for SLOT=5, as we'll get a 5.1 or 6 or something in March 2023 instead.

Reply:
As per IRC discussion, just go with `SLOT="4.1/0"` then